.circleci/config.yml

```diff
@@ -16,27 +16,27 @@ jobs:
       # Take advantage of latest cache of dependencies
       - restore_cache:
           keys:
-            - v1-dependencies-{{ checksum ".circleci/requirements.txt" }}
+            - v1-dependencies-{{ checksum "requirements.txt" }}
```

Comment:
what happened here? Do we just have one requirements.txt file now? I thought we intentionally had two: one for the scroller and one for circle ci... Although I never did quite understand why we needed pandas sql in both. Thoughts?

Reply:
My guess is that pandasql was installing pandas and numpy which and providing all the necessary dependencies for datascroller. Without pandasql there's no pandas and you would get an error. I could see us having a requirements for datascroller alone and then extra requirements for CircleCI but we'd have two install statements.